index.src.html

```diff
@@ -195,6 +195,11 @@
     "matrix" → {{VideoMatrixCoefficients/bt709}},
     "fullRange" → `false`]».
 
+: <dfn>Half Away From Zero</dfn>
```

Comment:
Editors Call: No need for this, just say "round". This is what we all understand "round" to mean by default.